simulation/tests/it/main.rs

```diff
@@ -0,0 +1,3 @@
+#![cfg(madsim)]
+
+mod curp;
```

Comment:
It looks weird to include useless files in this PR

Reply:
Same as above.